sites/diverseeducation.com/server/templates/static-pages/mppwsa-yearly.marko

```diff
@@ -239,7 +275,25 @@ $ const ideas = currentYearData ? currentYearData.ideas: title;
               </tbody>
             </if>
           </table>
-        </if>
+              <if(currentYearData && currentYearData.nationalPerspective)>
+                <br/>
+              <h2 class="page-wrapper__website-section-name">A NATIONAL PERSPECTIVE</h2>
+                <div class="page-wrapper__website-section-description">
+                <p>$!{currentYearData.nationalPerspective}</p><br/>    
+             <div class="row">
+            <div class="col" align="center">
+                    <marko-web-img
+                class=["lazyload"]
+                alt="US map"
+                src=currentYearData.mapSrc
+                srcset=`${currentYearData.mapSrc} x2`
+                lazyload=true
+              />
+              </div>
+             </div>
+           </div>   
+            </if>
+         </if>
```

Comment:
Check the tabbing of the tags here this doesn't appear to be correct (as in the tags aren't aligned inner to outer)